src/test/java/io/confluent/connect/elasticsearch/ElasticsearchWriterTest.java

```diff
@@ -324,7 +431,9 @@ private ElasticsearchWriter initWriter(JestClient client, boolean ignoreKey, Set
         .setLingerMs(1000)
         .setRetryBackoffMs(1000)
         .setMaxRetry(3)
+        .setBehaviorOnNullValues(behavior)
         .build();
+    behaviorOnNullValues = behavior;
```

Comment:
This doesn't do what we want. It's set after it's used in base class's `setUp`. That's the only place where `getBehaviorOnNullValues` and setting it here it's too late. 

Which also means that the test for the non-default options (delete and fail) is not testing what we want? (can't test thoroughly now, intellij is freaking out with jvm versions)

Reply:
Totally right. Turns out, `behaviorOnNullValues` doesn't have to be adjusted at all for the `DataConverter` used in the `ElasticsearchSinkTestBase` class and (for now) can and should just be set to `FAIL`.